h2o-genmodel/src/main/java/hex/genmodel/algos/glrm/GlrmMojoModel.java

```diff
@@ -25,11 +25,15 @@
   public GlrmInitialization _init;
   public int _ncats;
   public int _nnums;
-  public double[] _normSub;
-  public double[] _normMul;
+  public double[] _normSub; // used to perform dataset transformation.  When no transform is needed, will be 0
+  public double[] _normMul; // used to perform dataset transformation.  When no transform is needed, will be 1
   public long _seed;  // added to ensure reproducibility
   public boolean _transposed;
   public boolean _reverse_transform;
+  public double _accuracyEps = 1e-10; // reconstruction accuracy A=X*Y
+  public int _iterNumber = 200; // maximum number of iterations to perform X update.
+  public String _transform = "NONE";  // transform applied to user dataset
```

Comment:
I don't see `_transform` is redefined / set or used anywhere down from here. Do I miss something?

Reply:
Ruslan:

You did not miss anything but caught a bug for me.  I am deleting it now.  However, this does not affect anything and you do not need a new jar for this.  It is just good programming practice.


Thank you.